packages/dnb-eufemia/src/components/form-status/FormStatus.js

```diff
@@ -451,6 +451,7 @@ export default class FormStatus extends React.PureComponent {
 }
 
 export const ErrorIcon = (props) => {
+  const { title = 'error' } = props || {}
```

Comment:
```suggestion
  const { title = 'error' } = props
```

When calling a React component without any props, the component will still get an empty object (I'm 99.9% sure).

Reply:
I think props will be `undefined` 🤔 
That's why I had to add this fallback, `props || {}` 😂